modules/exploits/linux/http/nagios_xi_plugins_check_plugin_authenticated_rce.rb

```diff
@@ -89,7 +89,8 @@ module does not presently support exploiting these targets.
         'DefaultTarget' => 1,
         'Notes' => {
           'Stability' => [ CRASH_SAFE, ],
-          'SideEffects' => [ ARTIFACTS_ON_DISK, IOC_IN_LOGS, CONFIG_CHANGES ]
+          'SideEffects' => [ ARTIFACTS_ON_DISK, IOC_IN_LOGS, CONFIG_CHANGES ],
+          'Reliability' => [] # fixing rubocop issues
```

Comment:
```suggestion
          'Reliability' => []
```
I see what you mean, although I think this comment belongs more on the github PR rather than in the module itself. Either way, thanks!

Reply:
Yeah sure. I apologise for this from next time I will add it in the PR comment rather than in the code comments. I will remove this.